icgc.bionimbus.org/portal/gitops.json

```diff
@@ -18,7 +18,14 @@
   },
   "components": {
     "appName": "Bionimbus PDC for ICGC",
-    "index": {
+      "banner": [
+        {
+          "type": "warning",
+          "message": "The ICGC Site will undergoing maintenance on Friday, December 6, 2024, 8am-5pm CST.  Users may experience service interruptions during this time.", 
```

Comment:
Instead of calling it the ICGC site, can we modify to "Bionimbus Protected Data Cloud"? Thanks!

Reply:
Sure.